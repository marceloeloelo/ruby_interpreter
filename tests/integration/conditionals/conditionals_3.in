xyz = 5
if xyz > 4
  # puts 'La variable xyz es mayor que 4'
  # puts 'Puedo poner mas instrucciones dentro del if'
  if xyz == 5
    # puts 'Se puede anidar un bloque if, else, end dentro de otro'
  else
    # puts "Parte del bloque anidado"
  end
else
  # puts 'La variable xyz no es mayor que 5'
  # puts 'Tambien puedo poner multiples sentencias'
end
