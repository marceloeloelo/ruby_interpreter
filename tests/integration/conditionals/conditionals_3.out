La variable xyz es mayor que 4
Puedo poner mas instrucciones dentro del if
Se puede anidar un bloque if, else, end dentro de otro
0
